src/main/java/org/reactome/server/service/controller/citation/CitationController.java

```diff
@@ -0,0 +1,294 @@
+            }
+            map.put("pathwayTitle", p.getDisplayName());
+            map.put("hasImage", p.getHasDiagram());
+
+            List<HashMap<String, String>> authors = null;
+            List<InstanceEdit> instanceEdits = null;
+
+            // the authors field gets populated in the order of priority as defined by
+            // the if else-if conditions below
+            // this is because we have pathways with missing authors, creators and reviewers
+            // in case none of these are available, the `authors` field will have a null value and
+            // won't show up in the response at all
+            if (p.getAuthored() != null && !p.getAuthored().isEmpty()) {
+                instanceEdits = p.getAuthored();
+            } else if (p.getCreated() != null) {
+                instanceEdits = new ArrayList<>();
+                instanceEdits.add(p.getCreated());
+            } else if (p.getReviewed() != null && !p.getReviewed().isEmpty()) {
+                instanceEdits = p.getReviewed();
+            }
+
+            if (instanceEdits != null) {
+                authors = new ArrayList<>();
+                for (InstanceEdit instanceEdit : instanceEdits) {
+                    for (Person person : instanceEdit.getAuthor()) {
+                        HashMap<String, String> author = new HashMap<>();
+                        author.put("lastName", person.getSurname());
+                        author.put("initials", String.join(".", person.getInitial().split("")) + ".");
+                        author.put("firstName", person.getFirstname());
+                        author.put("fullName", person.getDisplayName());
+                        authors.add(author);
+                    }
+                }
+            }
+            map.put("authors", authors);
+            map.put("releaseVersion", generalService.getDBInfo().getVersion());
+        }
+        return ResponseEntity.ok(map);
+    }
+
+    // endpoint for getting the string citation for the download page
+    @GetMapping(value = "/download", produces = MediaType.TEXT_HTML_VALUE)
+    public ResponseEntity<String> downloadCitation() {
+        String downloadLink = "https://reactome.org/download-data/";
+        return ResponseEntity.ok("\"Name of file\", Reactome, " + generalService.getDBInfo().getVersion() + ", " + downloadLink);
+    }
+
+    // end point for getting data for citing any static citation, given the PMID
+    @GetMapping(value = "/static/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> staticCitation(@ApiParam(value = "PMID of the requested citation", required = true)
+                                                @PathVariable String id) throws Exception {
+        try {
+            HttpGet request = new HttpGet(new URIBuilder()
+                    .setScheme("https")
+                    .setHost("www.ebi.ac.uk")
+                    .setPath("/europepmc/webservices/rest/search")
```

Comment:
Since this endpoint is used twice in this class. You can create a static final together with the attributes.
`private static String EUROPE_PMC_ENDPOINT = "https://www.ebi.ac.uk/europepmc/webservices/rest/search";`
then in the `URIBuilder` you can use like `new URIBuilder(EUROPE_PMC_ENDPOINT)`